resources/decision_tree_samples.json

```diff
@@ -0,0 +1,85 @@
+{
+  "rootNode": "inheritance",
+  "nodes": {
+    "inheritance": {
+      "type": "BOOL",
+      "description": "Inheritance match",
+      "query": {
+        "field": "FORMAT/VIM",
+        "operator": "==",
+        "value": 1
+      },
+      "outcomeTrue": {
+        "nextNode": "denovo"
+      },
+      "outcomeFalse": {
+        "nextNode": "exit_r"
+      },
+      "outcomeMissing": {
+        "nextNode": "denovo"
+      }
+    },
+    "denovo": {
+      "type": "BOOL",
+      "description": "Inheritance Denovo",
+      "query": {
+        "field": "FORMAT/VID",
+        "operator": "==",
+        "value": 1
+      },
+      "outcomeTrue": {
+        "nextNode": "depth"
+      },
+      "outcomeFalse": {
+        "nextNode": "exit_r"
+      },
+      "outcomeMissing": {
+        "nextNode": "depth"
+      }
+    },
+    "depth": {
+      "type": "BOOL",
+      "description": "Depth",
+      "query": {
+        "field": "FORMAT/DP",
+        "operator": ">=",
+        "value": 20
+      },
+      "outcomeTrue": {
+        "nextNode": "exit_k"
+      },
+      "outcomeFalse": {
+        "nextNode": "exit_r"
+      },
+      "outcomeMissing": {
+        "nextNode": "exit_k"
+      }
+    },
+    "genotype_quality": {
+      "type": "BOOL",
+      "description": "Genotype quality",
```

Comment:
Genotype quality (GATK)

Reply:
is gewoon een VCF spec veld toch? niet specifiek GATK?

Reply:
you are right, my bad.